protostar/protostar_toml/protostar_contracts_section.py

```diff
@@ -50,3 +51,22 @@ def to_dict(self) -> "ProtostarTOMLSection.ParsedProtostarTOML":
             result[contract_name] = [str(path) for path in paths]
 
         return result
+
+    def get_contract_names(self) -> List[str]:
+        return list(self.contract_name_to_paths.keys())
+
+    def get_relative_contract_source_paths(self, contract_name: str) -> List[Path]:
+        self.assert_contract_is_defined(contract_name)
+        source_paths = self.contract_name_to_paths[contract_name]
+        return source_paths
+
+    def assert_contract_is_defined(self, contract_name: str):
```

Comment:
```suggestion
    def check_contract_is_defined(self, contract_name: str):
```

assert suggest it will raise `AssertionError` and can be turned off